site/topics/genetic-programming/genetic-programming.rst

```diff
@@ -154,10 +154,105 @@ Representation
 Language
 --------
 
+* The set of available operators and operands use for creating and modifying the trees is called the *language*
+* The operators and operands are something that can be adjusted as needed
+
+* With the breast cancer example, the language could be a collection of arithmatic and boolean operators and operands
+
+    * Operators
+
+        * Binary Operators
+
+            * :math:`+`
+            * :math:`-`
+            * :math:`\times`
+            * :math:`/`
+            * :math:`<`
+            * :math:`>`
+            * :math:`==`
+            * and
+            * or
+
+        * Unary Operators
+
+            * :math:`sin`
+            * :math:`cos`
+            * :math:`e`
+            * not
+
+
+    * Operands
+
+        * Constants (e.g. :math:`4`, True, False)
+        * Variables (e.g. size and shape)
+
+
+
+.. figure:: symbolic_regression_tree.png
+    :width: 333 px
+    :align: center
+
+    S-expression for the mathematical expression :math:`(1.2 - x) + sin((1.2 - x) \times y \times e^{x})`. This tree
+    contains arithmatic operators and operands (constants and variables).
+
+
+* When working with mathematical expressions, the language could be a collection of arithmatic operators and operands
+
+    * Operators
+
+        * Binary Operators
+
+            * :math:`+`
+            * :math:`-`
+            * :math:`\times`
+            * :math:`/`
+
+
+        * Unary Operators
+
+            * :math:`sin`
+            * :math:`cos`
+            * :math:`e`
+
+
+    * Operands
+
+        * Constants (e.g. :math:`1.2`)
+        * Variables (e.g. :math:`x` and :math:`y`)
```

Comment:
This duplication is unnecessary imo. Stating something like _numerical operations and operands above_ would be sufficient.